test/functions/MarshalUriFromSapiTest.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/**
+ * @see       https://github.com/zendframework/zend-diactoros for the canonical source repository
+ * @copyright Copyright (c) 2019 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   https://github.com/zendframework/zend-diactoros/blob/master/LICENSE.md New BSD License
+ */
+
+declare(strict_types = 1);
+
+namespace ZendTest\Diactoros\functions;
+
+use PHPUnit\Framework\TestCase;
+use function Zend\Diactoros\marshalUriFromSapi;
+
+class MarshalUriFromSapiTest extends TestCase
+{
+    /**
+     * @param string $httpsValue
+     * @param string $expectedScheme
+     * @dataProvider returnsUrlWithCorrectHttpSchemeFromArraysProvider
+     */
+    public function testReturnsUrlWithCorrectHttpSchemeFromArrays(string $httpsValue, string $expectedScheme) : void
+    {
+        $server = [
+            'HTTPS' => $httpsValue,
+            'SERVER_NAME' => 'localhost',
+            'SERVER_PORT'=>'80',
```

Comment:
missing spaces around `=>`

Reply:
> ... Thank you for noticing and investing the time into fixing a 10 month old bug. We really appreciate that. It would be easier for us to include the patch if we had a test that would catch a possible regression. So would you mind investing more time into adding one? And while you're at it, would you mind replacing the double quotes with single quotes and making sure that there is always one space around the '=>'? Thanks.

> ... Thanks for adding a test and for fixing the CodingStyle as well. I'Ve noticed one line that seems to have slipped your attention but don't worry, I'll fix that when I merge the PR.

Would we not actually need this PR merged ASAP for a project I would have stopped contributing after the first reply. And I can increasingly understand why people are moving – not away from ZF but – towards other frameworks. 

As maintainers we need to have a more welcome and thanking attitude even towards other maintainers. Yes. I know it takes time that we as maintainers put into it in out free time but that is not an excuse. Code is not everything.

Reply:
@heiglandreas thanks for your effort, finding the bug and preparing the PR. I really appreciate it. I am not the maintainer of the repository, I am just reviewing to get the PR ready for the maintainer to merge ASAP. You don't need to fix these changes, as everything was minor, and as you noted could be fixed on merge.

Sorry that my PR was short, just noted whatever was missing, without all "thankful" messages. I am concrete, that's it. Sorry.

Reply:
@webimpress **I** can live with that. But between all concretenes it will scare new contributors away. So if we as contributors want to attract more people to help out and fix stuff we need to make them feel more welcome. Otherwise they will go somewhere else. And tell others that it's not a good experience to contribute here...